modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -225,6 +225,13 @@ TEST_P(Test_ONNX_layers, Gather)
     testONNXModels("gather", npy, 0, 0, false, false);
 }
 
+TEST_P(Test_ONNX_layers, GatherElements)
```

Comment:
Could you also add a accuracy test here to test the raft model? Before this, you need to put your changes from https://github.com/opencv/opencv_extra/pull/1085 to https://github.com/opencv/opencv_extra/pull/1082 so that our CI system can detect the changes and then run the tests.

Reply:
Done